### DISCUSSION TITLE
How to See Node Neighborhood?

### DISCUSSION THREAD

**Discussion by kde-baskets-user** [jgraph/drawio-desktop#1526] [Q&A]
Hello. What's the best way to see a node neighborhood, i.e. all nodes connected to the node, regardless of direction. I have  
a pretty complicated diagram and have a hard time finding all neighbors. Also what about extended neighborhood, i.e. see all nodes connected to a given one through less than MAX hops?

Thanks.

**Comment by sivaswami** [comment]
This idea is similar to https://github.com/jgraph/drawio-desktop/discussions/1594
A Node search dialog to find neighboring nodes (1st level, 2nd level). 
or select nodes based on their colors  or shape
might be helpful
